src/main/java/uk/ac/cam/cl/dtg/segue/dto/content/QuestionDTO.java

```diff
@@ -39,10 +42,11 @@ public QuestionDTO() {
     }
 
     /**
-     * Gets the answer.
-     * 
+     * Gets the answer, defaults to not return to the user.
+     * Manually unignored e.g. quick questions
```

Comment:
This is a bit too terse to be clear. You mean it needs to be manually unignored for questions where the answer should be public.